awscli/examples/networkflowmonitor/get-query-status-monitor-top-contributors.rst

```diff
@@ -0,0 +1,13 @@
+**To retrieve the status of the query**
+
+The following ``get-query-status-monitor-top-contributors`` example displays the current status of the query in the specified account. ::
+
+    aws networkflowmonitor get-query-status-monitor-top-contributors \
+        --monitor-name Demo \
+        --query-id 5398eabd-bc40-3f5f-aba3-bcb639d3c7ca
+
+Output::
+
+    {
+        "status": "SUCCEEDED"
+    }
```

Comment:
Every example needs a for more information link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link